python/shotgun_model/shotgun_hierarchy_model.py

```diff
@@ -60,6 +70,8 @@ def __init__(self, parent, schema_generation=0, bg_task_manager=None):
         """
         super(ShotgunHierarchyModel, self).__init__(parent, bg_task_manager)
 
+        self._logger = sgtk.platform.get_logger(__name__)
```

Comment:
Yup, I was working around a limitation of our doc generating script, but I've now fixed it, so we're all good.